other/dns-policy-and-dns-config/dns-policy-and-dns-config.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: dns-policy
+  annotations:
+    policies.kyverno.io/title: Change DNS Config and Policy
+    policies.kyverno.io/category: Other
+    policies.kyverno.io/severity: medium
+    kyverno.io/kyverno-version: 1.7.3
+    kyverno.io/kubernetes-version: "1.23"
```

Comment:
Same with Kubernetes version.

Reply:
I tested it on the cluster with the 1.23 version.